src/plugins/Network.js

```diff
@@ -1,8 +1,13 @@
 'use strict';
 
+const log = require('loglevel');
+log.setDefaultLevel('debug');
```

Comment:
Do we need to keep this?

Reply:
we need it to log errors such:
`log.error('Selected profile not found');`